packages/perseus/src/interactive2/arrowhead.ts

```diff
@@ -0,0 +1,96 @@
+import {vector as kvector} from "@khanacademy/kmath";
+import _ from "underscore";
+
+import KhanMath from "../util/math";
+
+import {getClipPoint} from "./get-clip-point";
+import WrappedPath from "./wrapped-path";
+
+import type {Graphie} from "../util/graphie";
+import type {Coord} from "@khanacademy/perseus";
+
+export class Arrowhead extends WrappedPath {
+    private static scale = 1.4;
+    center: Coord;
+
+    constructor(graphie: Graphie, style: any) {
+        // Points that define the arrowhead
+        const center: Coord = [0.75, 0];
+        let points: Coord[] = [
+            [-3, 4],
+            [-2.75, 2.5],
+            [0, 0.25],
+            center,
+            [0, -0.25],
+            [-2.75, -2.5],
+            [-3, -4],
+        ];
```

Comment:
TODO: inline this value into the `_.map()` so `points` can be a const?

(while we're at it, we can convert `_.map()` to `Array.map()`.)

Reply:
I did wonder if we could even go further and define the arrow head as a simple SVG element and then only use `transform()` to place it.  But that might be too big of a refactor to be worth it.  :)